platform/linux-generic/include/odp_ipsec_internal.h

```diff
@@ -272,6 +272,13 @@ struct ipsec_sa_s {
 			} out;
 		};
 	} sa_info;
+
+	/*
+	 * Flag to check if the SA expiry status event was already
+	 * sent. This is applicable only for IPsec SA in inline outbound
+	 * mode.
+	 */
+	odp_atomic_u32_t expiry_notified;
```

Comment:
It would be good mention in the comment that this is only for the soft expiry status event. Perhaps the field could also be renamed to soft_expiry_notified?

Reply:
Ack. Will update with the next version.